graylog2-server/src/main/java/org/graylog2/shared/bindings/providers/OkHttpClientProvider.java

```diff
@@ -37,17 +37,20 @@
 import javax.inject.Named;
 import javax.inject.Provider;
 import javax.inject.Singleton;
+import javax.net.SocketFactory;
 import java.io.IOException;
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.Proxy;
 import java.net.ProxySelector;
+import java.net.Socket;
 import java.net.SocketAddress;
 import java.net.URI;
 import java.net.UnknownHostException;
 import java.util.List;
 import java.util.Locale;
 import java.util.Set;
+import java.util.function.Predicate;
```

Comment:
those imports can go and the diff should be empty

Reply:
Removed, thanks!